src/open_drive_parser.py

```diff
@@ -55,7 +57,7 @@ def __parse_lanes(self, lanes):
                     lane = rw.Lane(
                         int(att["id"]),
                         att["type"],
-                        bool(att["level"])
+                        utils.convertStringToBool(att["level"])
```

Comment:
Fixed bug - bool("string") will always return `True` if not empty string from before